src/content/platform-integration/web/wasm.md

```diff
@@ -33,6 +34,14 @@ Revisit this page for the latest updates.
 [`package:web`]: {{site.pub-pkg}}/web
 [`dart:js_interop`]: {{site.dart.api}}/{{site.dart.sdk.channel}}/dart-js_interop 
 
+:::note
+For information on how to optimize web loading speed,
+check out the (free) article on Medium,
+[Best practices for optimizing Flutter web loading speed][article].
+
+[article]: {{site.flutter-medium}}/best-practices-for-optimizing-flutter-web-loading-speed-7cc0df14ce5c
+:::
```

Comment:
**question:** As this is repeated in [another file](https://github.com/flutter/website/pull/10516/files#diff-40540badc69a9b7240fba3b99d24ebfff2735f812a45aa6d419199f33beb2986R15-R21), why not make this an include / render instead?